hack/list-images.sh

```diff
@@ -129,7 +129,7 @@ done
 # These services use raw manifests rather than Helm charts so list the images directly from the manifests.
 # If more raw manifest services are added, then they should be added to the list of paths below.
 gojq --yaml-input --raw-output 'select(.kind | test("^(?:Deployment|Job|CronJob|StatefulSet|DaemonSet)$")) |
-                                .spec.template.spec |
+                                (.spec.template.spec // .spec.jobTemplate.spec.template.spec) |
```

Comment:
now you need to add the missing `docker.io/bitnami/kubectl:1.29.6` image in license manifest as this change is picking up the missing image 😄

Reply:
done